class AlreadyExistsError(Exception):
    """Called if some object already exist"""
    pass
